resources/definitions/fdmprinter.def.json

```diff
@@ -4632,14 +4632,60 @@
                 "adhesion_extruder_nr":
                 {
                     "label": "Build Plate Adhesion Extruder",
-                    "description": "The extruder train to use for printing the skirt/brim/raft. This is used in multi-extrusion.",
+                    "description": "The extruder train to use for printing the skirt/brim. This is used in multi-extrusion.",
                     "type": "extruder",
                     "default_value": "0",
                     "value": "defaultExtruderPosition()",
-                    "enabled": "extruders_enabled_count > 1 and resolveOrValue('adhesion_type') != 'none'",
+                    "enabled": "extruders_enabled_count > 1 and resolveOrValue('adhesion_type') != 'none' and resolveOrValue('adhesion_type') != 'raft'",
                     "settable_per_mesh": false,
                     "settable_per_extruder": false
                 },
+                "raft_extruder_nr": 
```

Comment:
You could re-use `adhesion_extruder_nr` for this. There's no reason why it should be a separate setting, I think.